pantheon-bundle/frontend/src/app/versions.tsx

```diff
@@ -136,13 +136,13 @@ class Versions extends Component<IProps, IState> {
                 {this.state.publishAlertVisible && <div className="notification-container pant-notification-container-md">
                     <Alert
                         variant="warning"
-                        title="Module Versions"
+                        title="Publishing Document"
```

Comment:
Can we write instead as :

"Publishing" {this.props.contentType}  ?

Reply:
I tried the above suggestions. {this.props.contentType} or {...this.props.contentType} renders empty. I ended up creating a state variable to display the Alert title. That seems to reflect the contentType for publishing alerts.

Reply:
Looks cleaner :+1: